src/conda_devenv/devenv.py

```diff
@@ -467,18 +550,24 @@ def _call_conda(env_manager: Literal["conda", "mamba"], args: Sequence[str]) ->
     )
 
 
+def _get_target_env_name(args: argparse.Namespace, conda_yaml_dict: YAMLData) -> str:
+    return args.name or get_env_name_from_yaml_data(conda_yaml_dict)
+
+
 def write_activate_deactivate_scripts(
     args: argparse.Namespace,
     conda_yaml_dict: YAMLData,
     env_manager: Literal["conda", "mamba"],
-    environment: Environment,
     env_directory: Path | None,
 ) -> None:
     if env_directory is None:
-        env_name = args.name or get_env_name_from_yaml_data(conda_yaml_dict)
+        env_name = _get_target_env_name(args, conda_yaml_dict)
         env_directory = get_env_directory(env_manager, env_name)
         if env_directory is None:
-            raise UsageError(f"Couldn't find directory of environment '{env_name}'")
+            raise UsageError(
+                f"Could not find directory of environment '{env_name}' "
+                f"when trying to write activate scripts (should have been created by now)."
```

Comment:
I wonder why the linter didn't catch this:

```suggestion
                "when trying to write activate scripts (should have been created by now)."
```

Reply:
I think because to the parser it is the same string (the continuation causes a single string to be parsed).